client/src/components/DeleteModal/DeleteModal.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Modal from '../Modal';
+import Button from '../Button';
+import { alertActions, libraryActions } from '../../redux';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+class DeleteModal extends Component {
+  deleteEntity = (type, id) => {
+    const { libraryActions } = this.props;
+    libraryActions.remove(
+      id,
+      this.props.deleteSuccess,
+      this.props.deleteFailure,
+    );
+  };
+
+  render() {
+    const { deleteModal, closeDeleteModal, deleteEntity } = this.props;
+    return (
+      <Modal isOpen={true}>
+        <div className="library-search__delete-modal">
+          {/* Data type is sometimes stored as deleteModal.type and sometimes as deleteModal._original.type*/}
+          Are you sure you want to delete the{' '}
+          {deleteModal.type || deleteModal._original.type}{' '}
+          <i>{deleteModal.name}</i>?
+          <div>
+            <Button color="neutral" onClick={closeDeleteModal}>
+              No
+            </Button>
+            <Button
+              type="submit"
+              onClick={() =>
+                this.deleteEntity(deleteModal.type, deleteModal._original._id)
+              }
+            >
+              Yes
+            </Button>
+          </div>
+        </div>
+      </Modal>
+    );
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    alertActions: bindActionCreators({ ...alertActions }, dispatch),
```

Comment:
Since it isn't used, please remove `alertActions` from this file (both in this function and the imports at the top of the file).